src/components/ActionBar/style.css

```diff
@@ -0,0 +1,15 @@
+.base {
```

Comment:
So most all of these new style files are just cutting stuff out from `src/style/style.css` right?

Can you highlight any new style files that actually change anything substantial from `src/style/style.css` and need explicit reviewing?